packages/manager/src/features/Databases/DatabaseDetail/DatabaseSettings/DatabaseSettingsSuspendClusterDialog.tsx

```diff
@@ -0,0 +1,111 @@
+import { Engine } from '@linode/api-v4/lib/databases';
+import { DialogActions } from '@mui/material';
+import { useSnackbar } from 'notistack';
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+import { ActionsPanel } from 'src/components/ActionsPanel/ActionsPanel';
+import { Checkbox } from 'src/components/Checkbox';
+import { Dialog } from 'src/components/Dialog/Dialog';
+
+import { Notice } from 'src/components/Notice/Notice';
+import { Typography } from 'src/components/Typography';
+import { useSuspendDatabaseMutation } from 'src/queries/databases/databases';
+import { getAPIErrorOrDefault } from 'src/utilities/errorUtils';
+
+export interface SuspendDialogProps {
+  databaseEngine: Engine;
+  databaseID: number;
+  databaseLabel: string;
+  onClose: () => void;
+  open: boolean;
+}
+
+export const DatabaseSettingsSuspendClusterDialog: React.FC<SuspendDialogProps> = (
+  props
+) => {
```

Comment:
We prefer to not use `React.FC` in our repo. You can type your props this way:

```suggestion
export const DatabaseSettingsSuspendClusterDialog = (
  props: SuspendDialogProps
) => {
```

Reply:
Thanks! I've made this update.